core/src/test/java/org/hibernate/ogm/backendtck/optimisticlocking/OptimisticLockingTest.java

```diff
@@ -103,7 +104,7 @@ public void updatingEntityUsingOldVersionCausesException() throws Throwable {
 	 */
 	@Test
 	@SkipByGridDialect(
-			value = { HASHMAP, INFINISPAN, EHCACHE, NEO4J, COUCHDB, CASSANDRA, REDIS },
+			value = { HASHMAP, INFINISPAN, EHCACHE, NEO4J, COUCHDB, CASSANDRA, REDIS, REDIS_HASH },
```

Comment:
Redis/Redis Hash does not implement `OptimisticLockingAwareGridDialect`, there's no `UPDATE ... WHERE ...` command.